ReleaseProcess.md

```diff
@@ -9,7 +9,7 @@
    This script will warn you if any tickets are merged in that are not "Ready 
    to Deploy". It will automatically push the tag and commit.
 3. Check build has passed in Teamcity
-4. [Connect to the UAT machine and deploy there](staging/README.md)
```

Comment:
this actually just contains info about provisioning a staging machine, not about release